projects/client/RabbitMQ.Client/src/client/impl/AutorecoveringConnection.cs

```diff
@@ -183,15 +189,15 @@ private bool ManuallyClosed
             {
                 lock (m_eventLock)
                 {
-                    m_recordedBlockedEventHandlers.Add(value);
+                    m_recordedBlockedEventHandlers += value;
```

Comment:
How is this relevant to the goal of this PR?

Reply:
Resource usage. Lists of event handlers are being created regardless of being used or not. And for the purpose of what a multicast delegate can do.